GitIQT/Scenarios/CreateBranch.cs

```diff
@@ -0,0 +1,57 @@
+﻿namespace GitIQT
+{
+    internal class CreateBranch : IScenario
+    {
+        private string Response = string.Empty;
+
+        private string Answer = string.Empty;
+
+        // Ask user to create their own branch
+        public void AskPrompt()
+        {
+            // Ask user to create their own branch
+            Answer = $"git branch feature/feature-name";
+
+            var prompt = $"What git command do you need to type in to create a new branch from the dev branch? (Please name it feature/feature-name)";
```

Comment:
Allow 
![image](https://user-images.githubusercontent.com/26447240/214761490-84120140-1ffb-4bf5-b54c-ee7640e92d2b.png)
 to also work